### DISCUSSION TITLE
Mac mini M2 下无法导入预设文件，一直显示无法导入预设，导入0个预设。

### DISCUSSION THREAD

**Discussion by ashllll** [community/community#56244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Mac mini M2 下无法导入预设文件，一直显示无法导入预设，导入0个预设。在拓竹官方软件内导入预设在导出，接着在导入Orca则可以正常显示预设配置
![53421db840d9584ddb6acef6dcb07189](https://github.com/community/community/assets/49464678/f58ee172-2e4f-48ef-a4f4-97f5feda9115)

**Comment by ghost** [comment]
抱歉，我不是 Mac 用户，所以我不知道出了什么问题。 您可以向专业的 Mac 开发人员寻求帮助。 在这种情况下，您的问题与 GitHub 无关，并且似乎与代码托管在 GitHub 上的开源程序有关，所以请不要在这里问这个问题。

提醒您，请在论坛上使用英语。 大多数 GitHub 员工都不懂中文。 我不得不使用谷歌翻译给你这个中文文本。

<hr>

抱歉，我不是 Mac 用戶，所以我不知道出了什麼問題。您可以向專業的 Mac 開發人員尋求幫助。在這種情況下，您的問題與 GitHub 無關，並且似乎與代碼託管在 GitHub 上的開源程序有關，所以請不要在這裡問這個問題。

提醒您，請在論壇上使用英語。大多數 GitHub 員工都不懂中文。我不得不使用谷歌翻譯給你這個中文文本。

<hr>

Sorry, I'm not a Mac user, so I don't know what's wrong. You can seek help from a professional Mac developer. In this case, your question has nothing to do with GitHub, and seems to be about an open source program whose code is hosted on GitHub, so please don't ask it here.

Just a reminder, please use English on the forum. Most GitHub employees don't understand Chinese. I had to use Google Translate to get the Chinese response.
